packages/experiments/src/components/Shimmer/ShimmerRectangle/ShimmerRectangle.styles.ts

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import {
+  IStyle,
+  IStyleSet
+} from '../../../Styling';
+import { IStyleFunction } from '../../../Utilities';
+
+export interface IShimmerRectangle {
+
+}
+
+/**
+ * ShimmerRectangle component props.
+ */
+export interface IShimmerRectangleProps extends React.AllHTMLAttributes<HTMLElement> {
+  /**
+   * Optional callback to access the IShimmerRectangle interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IShimmerRectangle) => void;
+
+  /**
+   * Sets the height of the rectangle.
+   * @default 16px
+   */
+  height?: number;
+
+  /**
+   * Sets width of the elemnts.
```

Comment:
Specify the unit of value, i.e in percent

Reply:
Fixed.